torchbenchmark/e2e_models/hf_bert/__init__.py

```diff
@@ -65,7 +86,11 @@ def __init__(self, test, batch_size=None, extra_args=[]):
     
     def prep(self, hf_args):
         # Initialize the accelerator. We will let the accelerator handle device placement for us in this example.
-        accelerator = Accelerator(fp16=(self.tb_args.fp16 == "amp"))
+        if hasattr(hf_args, "deepspeed_plugin"):
+            accelerator = Accelerator(deepspeed_plugin=hf_args.deepspeed_plugin, mixed_precision='bf16')
+        else:
+            # TODO(whc) restore this to previous state, using fp16?
+            accelerator = Accelerator(mixed_precision='bf16')
```

Comment:
`tb_args` is for something we want to control globally for all the models, since models are using different names of arguments for the same thing. For example, model A may use `--precision`, model B may use `--fp16`, to control the dtype of data they use. In this case, we use `tb_args.fp16` (sorry I think it is a bad naming from me, `tb_args.precision` is probably better) to specify it in the upper level, and tweak the actual configs for models case-by-case.

Do you think precision is something we want to have a universal name to config over all models? I think both precision and batch sizes config should have a global name over all future models we add.

Reply:
batch size definitely.

precision maybe, (precision is a better name for the arg).  It may still be debatable if one cmdline arg is enough detail for different ways a model may apply a given flag.  like, even if we pick bf16 for one model, it could use it for more ops or less ops, explicitly, or via AMP, etc.

i think the question is what does the benchmark user want to control?  definitely 32 vs 16 bit.  less definitely bf16 vs fp16, as which one is appropriate for the model may depend on other things.  

also, if you want to do a sweep: do you expect users to do a separate sweep with some subset of models with bf16 setting, and another one with a different set with fp16?  or do you want to get one set of results for all the 16 bit models?

Reply:
We don't have many models rn, so I think either way is fine to me. Let's remove the `--fp16` option for `tb_args` for now and always use `bf16`, since it is default.

@mrshenli what is your opinion here? Do you think people will be interested in comparing fp32 vs fp16 performance on different distributed frameworks (e.g. ddp vs fsdp)?